source/mongodb-handshake/handshake.rst

```diff
@@ -233,12 +276,6 @@ will return an error if these limits are not adhered to, which will result in
 handshake failure. Drivers MUST validate these values and truncate driver
```

Comment:
I think we should encourage driver devs to truncate the _platform_ field only. The driver name and version fields are too high value to truncate. Encourage implementers to only put high value information in the field (done't dump compiler and linker flags there) and place the highest value data at the beginning of the string.

Reply:
Relates to my comment in https://github.com/mongodb/specifications/pull/687#discussion_r345938767.

Reply:
added a section about this